service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -90,31 +92,16 @@ public ResponseEntity<RunStateResponse> postRun(String workflowUrl, Object workf
 
     ApiClient client = new ApiClient();
     client.setBasePath(this.cromwellConfig.baseUri());
-    WorkflowsApi workflowsApi = new WorkflowsApi(client);
+    Ga4GhWorkflowExecutionServiceWesAlphaPreviewApi wesApi =
+        new Ga4GhWorkflowExecutionServiceWesAlphaPreviewApi(client);
     String runId = UUID.randomUUID().toString();
 
     try {
-      workflowsApi.submit(
-          "v1",
-          null,
-          workflowUrl,
-          null,
-          workflowParams.toString(),
-          null,
-          null,
-          null,
-          null,
-          null,
-          null,
-          null,
-          null,
-          null,
-          null,
-          null);
+      wesApi.runWorkflow(workflowParams.toString(), null, null, null, null, workflowUrl, null);
```

Comment:
Would you need to update the Cromwell client with changes from [PR 6837](https://github.com/broadinstitute/cromwell/pull/6837) before merging this PR?

Reply:
I don't think so. The problem was the way that Wes was submitting the workflows on the cromwell end, so nothing on the client side would need to be updated that I'm aware of.